app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/218755863-724bbf0b-e4d4-41af-b211-7221e695a446.png)
fix this, please